hack/local-up-cluster.sh

```diff
@@ -813,6 +817,10 @@ EOF
       done
     fi >>/tmp/kube-proxy.yaml
 
+    if [[ "${REUSE_CERTS}" != true ]]; then
+        generate_kubeproxy_certs
```

Comment:
this change isn't described in the commit or PR description, what's going on here?

Reply:
Yes, I will update the PR with this change. but basically if we don't run the kobeproxy with don't need to generate it certificates. It similar change that I did in this commit  https://github.com/kubernetes/kubernetes/pull/74269/commits/33a81a7882d71e5d8f4a3850947c1d87677bf783 for kubelet. In general I think it will be better if we move the generate certificates to each method that start the service 
so that apiserver, scheduler kubeproxy kubelet will generate certificates as well.